client/src/components/approvals/PlanningApprovalSteps.js

```diff
@@ -0,0 +1,69 @@
+import Fieldset from "components/Fieldset"
```

Comment:
While it is a great idea to make a class for the approval steps, `PlanningApprovalSteps.js` and `PublicationApprovalSteps.js` are almost identical. I suggest to make a single generic class for both and instantiate it twice with whatever needs to be different.